micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/discovery/ZookeeperDependencyStubsConfigurationFix.groovy

```diff
@@ -0,0 +1,47 @@
+package com.ofg.infrastructure.discovery
+
+import com.ofg.config.BasicProfiles
+import org.springframework.beans.BeansException
+import org.springframework.beans.factory.config.BeanPostProcessor
+import org.springframework.cloud.zookeeper.discovery.dependency.StubsConfiguration
+import org.springframework.cloud.zookeeper.discovery.dependency.ZookeeperDependencies
+import org.springframework.cloud.zookeeper.discovery.dependency.ZookeeperDependency
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+
+/**
+ * Workaround for bug in spring-cloud-zookeeper
+ * https://github.com/spring-cloud/spring-cloud-zookeeper/issues/138
+ */
+@Configuration
+@Profile(BasicProfiles.SPRING_CLOUD)
+class ZookeeperDependencyStubsConfigurationFix implements BeanPostProcessor {
+
+    @Override
+    Object postProcessBeforeInitialization(Object bean, String beanName) throws BeansException {
+        if (bean instanceof ZookeeperDependencies) {
+            ZookeeperDependencies zookeeperDependencies = (ZookeeperDependencies) bean
+            zookeeperDependencies.getDependencyConfigurations().each { ZookeeperDependency dependency ->
+                if (!dependency.stubs) {
+                    ServicePath servicePath = new ServicePath(dependency.path)
+                    MicroserviceConfiguration.Dependency.StubsConfiguration stubConfiguration =
+                            new MicroserviceConfiguration.Dependency.StubsConfiguration(servicePath)
+
+                    dependency.stubs = stubConfiguration.toColonSeparatedDependencyNotation()
+                    dependency.stubsConfiguration = new StubsConfiguration(
+                            stubConfiguration.stubsGroupId,
+                            stubConfiguration.stubsArtifactId,
+                            stubConfiguration.stubsArtifactId
```

Comment:
That's mistake, but all in all `stubsConfiguration` is overridden during `ZookeeperDependencies` post construct. Removing this line.